pkg/cluster/cluster.go

```diff
@@ -0,0 +1,12 @@
+package cluster
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Cluster struct {
+	OperatorNamespace string
+	Client            client.Client
+	Cache             cache.Cache
+}
```

Comment:
I already mentioned it in another comment: https://github.com/codeready-toolchain/host-operator/pull/548#discussion_r759920112
this could be simplified
```suggestion
type Cluster struct {
	runtimecluster.Cluster
	OperatorNamespace string
}
```

Reply:
no, I prefer keeping it like this so it's easier to configure in the tests.